packages/one-app-bundler/webpack/module/webpack.client.js

```diff
@@ -60,7 +60,7 @@ module.exports = (babelEnv) => {
       performance: {
         maxAssetSize: configOptions.performanceBudget || 250e3,
         maxEntrypointSize: configOptions.performanceBudget || 250e3,
-        hints: process.env.NODE_ENV === 'development' ? 'warning' : 'error',
+        hints: babelEnv !== 'legacy' && (process.env.NODE_ENV === 'development' ? 'warning' : 'error'),
```

Comment:
This looks good. Another item we were trying to figure out was how to have the performance budget not consider the legacy build as well.